pulsar-client-cpp/docker/build-wheels.sh

```diff
@@ -25,6 +25,7 @@ BUILD_IMAGE_NAME="${BUILD_IMAGE_NAME:-apachepulsar/pulsar-build}"
 
 ROOT_DIR=$(dirname $0)/../..
 cd $ROOT_DIR
+ROOT_DIR=$(pwd)
```

Comment:
I don't this would have any effect since the variable is already set to the same value.

Reply:
Sorry, I forgot to upload the context of this issue. In the build process of python-client, Docker will reject the relative path when mounting volumes, i.e. 
```
Digest: sha256:2f75d7ba0cec73550deaa8a850fb3dcfcc47dd5aa7d17ad6fab3d4bae5ae5e7c
201
Status: Downloaded newer image for apachepulsar/pulsar-build:manylinux-cp27-cp27mu
202
docker: Error response from daemon: create pulsar-client-cpp/docker/../..: "pulsar-client-cpp/docker/../.." includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intended to pass a host directory, use absolute path.
203
See 'docker run --help'.
```

Reply:
Oh, gotcha. Can you add it as a comment in the script itself, so that it will not get accidentally removed later?

Reply:
Done.